src/test/java/uk/org/okapibarcode/output/SvgRendererTest.java

```diff
@@ -211,20 +211,41 @@ public void testEan13WithAddOn() throws IOException {
         ean.setQuietZoneHorizontal(5);
         ean.setQuietZoneVertical(5);
         ean.setContent("123456789012+12345");
-        test(ean, 2, Color.WHITE, Color.BLACK, "ean-13-with-add-on.svg", true);
+        test(ean, 2, Color.WHITE, Color.BLACK, "ean-13-with-add-on.svg", true, 0);
     }
 
     @Test
     public void testCode93With1dot2Magnification() throws IOException {
         Code93 code93 = new Code93();
         code93.setContent("123456789");
-        test(code93, 1.2, Color.WHITE, Color.BLACK, "code93-with-magnification-1.2.svg", true);
+        test(code93, 1.2, Color.WHITE, Color.BLACK, "code93-with-magnification-1.2.svg", true, 0);
+    }
+
+    @Test
+    public void testCode93Orientation90() throws IOException {
+        Code93 code93 = new Code93();
+        code93.setContent("123456789");
+        test(code93, 1.2, Color.WHITE, Color.BLACK, "code93-with-orientation-90.svg", true, 90);
+    }
+
+    @Test
+    public void testCode93Orientation180() throws IOException {
+        Code93 code93 = new Code93();
+        code93.setContent("123456789");
+        test(code93, 1.2, Color.WHITE, Color.BLACK, "code93-with-orientation-180.svg", true, 180);
+    }
+
+    @Test
+    public void testCode93Orientation270() throws IOException {
```

Comment:
The SVG previews on GitHub are showing that for 90 and 270 degree rotations we are fitting a vertical barcode into an image sized like a horizontal barcode. I think we need to be swapping the image width and height for these rotations.

GitHub doesn't show EPS previews, but I assume these files will have the same problem.